security/tor-devel/Portfile

```diff
@@ -4,7 +4,7 @@ PortSystem          1.0
 
 name                tor-devel
 conflicts           tor
-version             0.4.0.5
+version             0.4.1.1-alpha
```

Comment:
Is it a good idea to be using an alpha release ?

Reply:
I think it's fine. This is the development branch and we have been using alpha releases in the past. Users who wish to use stable releases should be on the regular tor port, not tor-devel.